packages/atkmm16.rb

```diff
@@ -0,0 +1,45 @@
+require 'package'
+
+class Atkmm16 < Package
```

Comment:
Why not `class Atkmm16 < Meson`?

Reply:
Because it has an extra `-Dbuild-documentation=false` line that should go in `meson_options`, but `meson_options` for the `buildsystems/meson` builds is currently broken.

Reply:
See https://github.com/chromebrew/chromebrew/issues/8445